Tests/Contoso.Test.Functional.Droid/MainActivity.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Reflection;
+using Android.App;
+using Android.OS;
+using Android.Util;
+using Xunit.Runners.ResultChannels;
+using Xunit.Runners.UI;
+
+namespace Contoso.Test.Functional.Droid
+{
+    [Activity(Label = "xUnit Android Runner", MainLauncher = true, Theme = "@android:style/Theme.Material.Light")]
+    public class MainActivity : RunnerActivity
+    {
+        protected override void OnCreate(Bundle bundle)
+        {
+            // tests can be inside the main assembly
+            AddTestAssembly(Assembly.GetExecutingAssembly());
+
+            //AddTestAssembly(typeof(PortableTests).Assembly);
+            // or in any assembly that you load (since JIT is available)
+
+            // Try to send results to the host via a socket for CI.
+            try
+            {
+                ResultChannel = TrxResultChannel.CreateTcpTrxResultChannel("10.0.2.2", 16384).Result;
```

Comment:
Should we move it to some config or to constant at least?

Reply:
+1 for moving to constant as 16384 port is used by both android and iOS.